server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/SessionOperations.java

```diff
@@ -30,4 +32,21 @@ public interface SessionOperations {
 
     void killQuery(@NonNull Connection connection, @NonNull String connectionId);
 
+    /**
+     * Get kill query SQL by connectionId
+     *
+     * @param connectionIds
+     * @return the map of connectionId to kill query SQL
+     */
+    Map<String, String> getKillQuerySqls(@NonNull List<String> connectionIds);
+
+
+    /**
+     * Get kill session SQL by connectionId
+     *
+     * @param connectionIds
+     * @return the map of connectionId to kill session SQL
+     */
+    Map<String, String> getKillSessionSqls(@NonNull List<String> connectionIds);
```

Comment:
actually, input as list, but output as map, there maybe some question for me.
1. if output map key is connectionId, will input list guarantee connectionIds will not duplicate? if so , set is more suitable
2. if batch is implement by single query, provide single query method is needed.

Reply:
👍🏻 Nice suggestions. I create two single methods:  `getKillQuerySql` and `getKillSessionSql`. The batch method will have a default implementation in the interface which relies on these two single methods. And the batch methods' input has been changed to a `Set`.